### PR TITLE
Add dedicated 'metagenome' organism directory with metagenome related files

### PR DESCRIPTION
**Pull request by jfy133**
I'm increasingly coming across cases where the current test-data 'organism' structure does not work with metagenomic tools, such as the repeated `genome.fasta` files causing name conflicts.

Furthermore, as metagenomics are dealing with multiple organisms, I need combinations of multiple organisms e.g. with taxonomies.

This PR adds a dedicated metagenome folder that provides a space for 'minimal' metagenomes related files of just 2 organisms, including compatible names.

It also updates the README to include missing entries, where I know what they are...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -560,6 +567,31 @@ The folder is structured in the following way: Any nonspecific-pangenome file is
     - nanopore
       - fastq
         - 'test.fastq.gz' synthetic raw long-read sequencing reads of the genome of the mammalian-gut-residing _Bacteroides fragilis_ YCH46 bacterium (NC_006347). Originally generated for the [MAG pipeline test dataset](https://github.com/nf-core/test-datasets/tree/mag).
+  - candidatus_portiera_aleyrodidarum
+    - genome
+      - ???
```
Comment (maxulysse) [inline README.md:572]: ???

File: README.md [hunk 2]
```diff
@@ -560,6 +567,31 @@ The folder is structured in the following way: Any nonspecific-pangenome file is
...
+    - illumina
+      - ???
```
Comment (maxulysse) [inline README.md:574]: ???

File: README.md [hunk 3]
```diff
@@ -560,6 +567,31 @@ The folder is structured in the following way: Any nonspecific-pangenome file is
...
+    - nanopore
+      - ???
```
Comment (maxulysse) [inline README.md:576]: ???

**Review by maxulysse — APPROVED** [review]

**Review by FriederikeHanssen — APPROVED** [review]
Looks very reasonable to me. We should just make sure somehow we don't have the fastas etc somewhere twice.

File: README.md (hunk 1 above)
Comment (jfy133) [reply README.md:572]: I just don't know about them unfortunately... I vaguely remember they may have come from bactopia @rpetit3 ?

**Comment by jfy133** [comment]
> Looks very reasonable to me. We should just make sure somehow we don't have the fastas etc somewhere twice.

These are technically there twice :\, but `gneome.fasta` twice  doesn't work well when trying to stage them together in one directory :grimacing:

**Comment by jfy133** [comment]
Ok I have the two :heavy_check_mark: so proceeding even if it's sarek focused :eyes:

**Pull request merged**